usermods/sd_card/sd_card.cpp

```
@@ -11,11 +11,35 @@
+    #define UM_SD_POCI 44
+  #else
+    #define UM_SD_POCI 36
+  #endif
+#endif
+#ifndef UM_SD_PICO
+  #define UM_SD_PICO 15
+#endif
+
+
+#ifndef WLED_USE_SD_MMC
+  // SD_MMC configuration handled elsewhere
```

Cppcheck (2.21.0)

[error] `#error` Platform not supported

(preprocessorErrorDirective)